### PR TITLE
Display network name for token rows

### PR DESCRIPTION
**Pull request by Jon-edge**
Screenshot 2023-08-30 at 4 28 41 PM
Screenshot 2023-08-30 at 4 29 26 PM
Screenshot 2023-08-30 at 4 30 27 PM

### CHANGELOG

- added: Network name display on token row UI

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205308893333756

### CODE REVIEW

File: src/components/data/row/CurrencyRow.tsx [hunk 1]
```diff
@@ -62,6 +62,9 @@ const CurrencyRowComponent = (props: Props) => {
   if (showTokenNames && tokenFromId != null) {
     displayCurrencyCode = `${tokenFromId.displayName}`
   }
+  if (token != null) {
```
Comment (paullinator) [inline src/components/data/row/CurrencyRow.tsx:65]: Do not add network names to Wallet List scene. There's not enough space and it was not part of the task

File: src/components/themed/CreateWalletSelectCryptoRow.tsx [hunk 2]
```diff
@@ -38,7 +38,9 @@ export const CreateWalletSelectCryptoRowComponent = (props: Props) => {
 
   const account = useSelector(state => state.core.account)
   const currencyConfigs = useWatch(account, 'currencyConfig')
-  const { currencyCode } = tokenId != null ? currencyConfigs[pluginId].allTokens[tokenId] : currencyConfigs[pluginId].currencyInfo
+  const isToken = tokenId != null
+  const { currencyCode } = isToken ? currencyConfigs[pluginId].allTokens[tokenId] : currencyConfigs[pluginId].currencyInfo
```
Comment (paullinator) [inline src/components/themed/CreateWalletSelectCryptoRow.tsx:42]: Do not destruction currencyCode immediately. Check if `currencyConfigs[pluginId].allTokens[tokenId]` is non-null or you might crash

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/data/row/CurrencyRow.tsx (hunk 1 above)
Comment (Jon-edge) [reply src/components/data/row/CurrencyRow.tsx:65]: The task mentioned "Search wallets list," what was the intended change?

File: src/components/themed/CreateWalletSelectCryptoRow.tsx (hunk 2 above)
Comment (Jon-edge) [reply src/components/themed/CreateWalletSelectCryptoRow.tsx:42]: Breaks the hook usage below. Will add another check

**Review by paullinator — APPROVED** [review]

**Pull request merged**